frontend/src/views/Main/components/Sidebar/components/ChatListItem/ChatListItem.tsx

```diff
@@ -1,16 +1,33 @@
+import { useNavigate } from "react-router-dom";
 import styles from "./ChatListItem.module.css";
 import ProfileIcon from "./components/ProfileIcon/ProfileIcon";
+import { socket } from "../../../../../../context/socket";
+import { useContext } from "react";
+import { socketContext } from "../../../../../../context/SocketContext";
+
+const ChatListItem = ({ name }: { name: string }) => {
```

Comment:
Let's move this props type into its own type above called Props so it's cleaner. I know right now it is only one prop but it could get pretty long and unruly and hard to read